calcom/cal.com
# Add reschedule reason to DB and emails

## What does this PR do?



This PR adds the rescheduling reason to the DB and displays it in the emails

Reschedule page
![image](https://user-images.githubusercontent.com/65426560/170735312-b9b8fe20-98d0-4f07-9f72-3a570724507e.png)

Reschedule email
![image](https://user-images.githubusercontent.com/65426560/170735807-89890df6-194d-48d3-8778-647c3e827cc2.png)

Request reschedule email
![image](https://user-images.githubusercontent.com/65426560/170735976-c87dae31-848c-417d-bed9-4ae61194dc95.png)

Fixes #2709 



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Create a booking
- Reschedule the booking, on the booking page it should look like the above screenshot
- When rescheduling enter a reason
- On the emails, check that the reason is added to the email
- Request to reschedule a booking and add a reason
    - The emails should display the reason


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
